src/types.ts

```diff
@@ -0,0 +1,71 @@
+// AUTOGENERATED - Use `utils/export_all.js` to regenerate.
+
+export * from './compat.d.js';
```

Comment:
is it meant to be re-generated manually or do we want it to be part of the build step?